cpp/src/cider/exec/nextgen/operators/ColumnToRowNode.cpp

```diff
@@ -104,6 +107,79 @@ class ColumnReader {
     }
   }
 
+  void readVariableSizeArrayCol(bool for_null) {
+    auto&& [batch, buffers] = context_.getArrowArrayValues(expr_->getLocalIndex());
+    utils::VarSizeArrayExprValue varsize_values(buffers);
+
+    auto& func = batch->getParentJITFunction();
+
+    if (for_null) {
```

Comment:
@Ma-Jian1 for this `for_null` path, is it always the same handle no matter what's the data type is?

Reply:
Per offline confirmation, this code path will be removed so we don't need consider for now.